### PR TITLE
feature#28 | added discord link to homepage

### PR DESCRIPTION
**Pull request by sachin9164**
Issue : #28 

Added discord link into "connect with us" section in JSLovers Homepage.

Screen Shot

Screenshot 2023-10-30 at 8 31 54 AM

### CODE REVIEW

**Comment by Neha** [comment]
Hey @sachin9164 , I have shared the discord link in the Issue. Please update and the screenshot is broken can you please check that?

**Comment by sachin9164** [comment]
@Neha updated link , added new screen shot please review the PR

File: index.html [hunk 1]
```diff
@@ -189,14 +189,22 @@ <h2 class="font-work-sans font-semibold mb-5 text-3xl">
...
                     src="resources/assets/website.png"
                     alt="jslovers twitter website"
                     class="hover:invert mx-auto"
                 />
                 <p class="mt-2 underline">Website</p>
             </a>
+            <a href="https://discord.com/invite/89QtcRw" target="_blank" class="hover:text-white inline-block">
+                <img
+                    src="resources/assets/discord.png"
+                    alt="jslovers twitter website"
```
Comment (Neha) [inline index.html:203]: Can you please keep the alt tag empty here? alt=""

Comment (sachin9164) [reply index.html]: Removed alt tag

**Pull request merged**